app/src/main/java/com/mertceyhan/bitcoinmarket/widget/ui/BitcoinWidget.kt

```diff
@@ -0,0 +1,124 @@
+package com.mertceyhan.bitcoinmarket.widget.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.glance.*
+import androidx.glance.action.actionStartActivity
+import androidx.glance.action.clickable
+import androidx.glance.appwidget.action.actionRunCallback
+import androidx.glance.layout.*
+import androidx.glance.text.FontWeight
+import androidx.glance.text.Text
+import androidx.glance.text.TextStyle
+import androidx.glance.unit.ColorProvider
+import com.mertceyhan.R
+import com.mertceyhan.bitcoinmarket.features.MainActivity
+import com.mertceyhan.bitcoinmarket.widget.callback.MarketRefreshCallback
+
+@Composable
+fun BitcoinWidget(currentPrice: String?, changeRate: String?, isChangeRatePositive: Boolean?) {
```

Comment:
Could you add preview versions for `BitcoinWidget`, please?

Reply:
Actually , I tried to create in many times but preview is not appear a round. I think it is because of glance api. Probably, it will be fixed with next releases.